### ISSUE TITLE
Same channel listed twice for import when using token

### ISSUE THREAD

**Issue by radinamatic** [learningequality/kolibri#6277]
### Observed behavior
After switching to import full channels, *and* inserting the token, if the user inserts token again, the channel will be listed two times in the list of channels. Stemmed from [this convo after the last week  bugbash](https://www.notion.so/learningequality/After-switching-to-import-full-channels-and-inserting-the-token-user-is-still-seeing-a-long-list-5caed6f604a34bb98395e794afd08cb3).

![2019-12-10_23-57-28](https://user-images.githubusercontent.com/1457929/70577653-0460d080-1bac-11ea-97ed-854c73536f7e.png)

### Expected behavior
Channel listed only once.
Ideally we would return a notification like `Channel already listed` or similar, but since we cannot intro new strings, just fixing this will be enough for now.

### User-facing consequences
User confusion.

### Errors and logs
…

### Steps to reproduce
Switch to import full channels and insert the same token twice.

### Context

 * Kolibri version: 0.13.0b4 & b5
 * Operating system: Windows 7
 * Browser: Firefox

**Comment by mrpau-richard** [comment]
I reproduced this issue on Kolibri mac app b4
![Screen Shot 2019-12-11 at 2 18 10 PM](https://user-images.githubusercontent.com/8663934/70596578-3bbd9480-1c21-11ea-84dc-2936f7d108a5.png)

**Comment by jonboiser** [comment]
To fix this for now, we can just treat the "new" unlisted channel from the modal as if it were added to the Available Channels List, by moving it to the top and giving it the green "New" badge, but not adding duplicates of that channel's row.

In the future, maybe we could put that "already installed" notification inside the modal or somewhere else.
